Tests/Integration/Core/OrderHelperTest.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use OxidEsales\Eshop\Application\Model\User;
+use OxidEsales\Eshop\Core\Registry;
+
+use Wirecard\Oxid\Core\OrderHelper;
+
+class OrderHelperTest extends \Wirecard\Test\WdUnitTestCase
+{
+    protected function dbData()
+    {
+        return [
+            [
+                'table' => 'oxuser',
+                'columns' => [
+                    'OXID',
+                    'OXACTIVE',
+                    'OXUSERNAME',
+                    'OXFNAME',
+                    'OXLNAME',
+                    'OXCOMPANY',
+                    'OXSTREET',
+                    'OXSTREETNR',
+                    'OXZIP',
+                    'OXCITY',
+                    'OXCOUNTRYID',
+                    'OXSTATEID',
+                ],
+                'rows' => [
+                    ['User ID 1', 1, 'User 1', 'User', 'One', 'Red Company', 'Blue Square', '1', '5555', 'Green City', '1', '1'],
```

Comment:
Please, split these values into multiple lines to stay consistent with the formatting below.